frontend/src/components/_generic/parts/DraggableSubWindow.css

```diff
@@ -4,7 +4,7 @@
     background-color: white;
     display: flex;
     flex-direction: column;
-    z-index: 30;
+    z-index: 3000;
```

Comment:
If you haven´t set the z-index anywhere else on this page/component etc. ,
you can set the z-index to 99/100...  3000 is okay but not really needed.